certbot/tests/cert_manager_test.py

```diff
@@ -179,7 +179,9 @@ def test_certificates_no_files(self, mock_utility, mock_logger):
         self.assertTrue(mock_utility.called)
         shutil.rmtree(tempdir)
 
-    def test_report_human_readable(self):
+    @mock.patch('certbot.cert_manager.ocsp.RevocationChecker.ocsp_status')
```

Comment:
Yeah, that form works too; it walks in via `cert_manager`'s reference to `ocsp`.